## To cite ChIP-Atlas in your publication:

- Paper
  - Shinya Oki, Tazro Ohta, et al. ChIP‐Atlas: a data‐mining suite powered by full integration of public ChIP‐seq data. EMBO Rep. (2018) e46255; doi: https://doi.org/10.15252/embr.201846255
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

## Publications citing ChIP-Atlas:

1. I. Abugessaisa et al., refTSS: A Reference Dataset for Human and Mouse Transcription Start Sites. __J. Mol. Biol.__ (2019), [Link](https://doi.org/doi:10.1016/j.jmb.2019.04.045).
2. Ö. Åkerborg et al., High-Resolution Regulatory Maps Connect Vascular Risk Variants to Disease-Related Pathways. __Circ. Genomic Precis. Med.__ 12, e002353 (2019), [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
3. M. Ando et al., Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers. __Nat. Commun.__ 10, 2188 (2019), [Link](https://doi.org/doi:10.1038/s41467-019-09937-w).
4. E. I. Bakhmet et al., hnRNP-K targets open chromatin in mouse embryonic stem cells in concert with multiple regulators. __Stem Cells__ (2019), [Link](https://doi.org/doi:10.1002/stem.3025).
5. S. Banerjee et al., Identifying transcriptional regulatory modules among different chromatin states in mouse neural stem cells. __Front. Genet.__ 10, 731 (2019), [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
6. M. Bocci et al., Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__. 22, 117–131 (2019), [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
7. C. Bravo González-Blas et al., cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. __Nat. Methods__. 16, 397–400 (2019), [Link](https://doi.org/doi:10.1038/s41592-019-0367-1).
8. E. Czipa et al., ChIPSummitDB: A ChIP-seq-based database of human transcription factor binding sites and the topological arrangements of the proteins bound to them. __bioRxiv__, 652420 (2019), [Link](https://doi.org/doi:10.1101/652420).
9. D. Delicado, B. Arconada, A. Aguado, M. A. Ramos, Multilocus phylogeny, species delimitation and biogeography of Iberian valvatiform springsnails (Caenogastropoda: Hydrobiidae), with the description of a new genus. __Zool. J. Linn. Soc.__ (2019), [Link](https://doi.org/doi:10.1093/zoolinnean/zly093).
10. K. Horie et al., Down-regulation of GATA1-dependent erythrocyte-related genes in the spleens of mice exposed to a space travel. __Sci. Rep.__ 9, 7654 (2019), [Link](https://doi.org/doi:10.1038/s41598-019-44067-9).
11. Y. Ishijima et al., The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Mol. Cell. Endocrinol.__ 483, 39–49 (2019), [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
12. T. Kehl et al., The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int. J. Cancer__. 144, 1432–1443 (2019), [Link](https://doi.org/doi:10.1002/ijc.31834).
13. M. Kojima et al., The histone methyltransferase WHSC1 is regulated by EZH2 and is important for ovarian clear cell carcinoma cell proliferation. __BMC Cancer__. 19, 455 (2019), [Link](https://doi.org/doi:10.1186/s12885-019-5638-9).
14. B. Krug et al., Pervasive H3K27 Acetylation Leads to ERV Expression and a Therapeutic Vulnerability in H3K27M Gliomas. __Cancer Cell__. 35, 782-797.e8 (2019), [Link](https://doi.org/doi:10.1016/j.ccell.2019.04.004).
15. A. S. Labade, A. Salvi, K. Karmodiya, K. Sengupta, Nup93 modulates spatiotemporal dynamics and function of the HOXA gene cluster during differentiation. __bioRxiv__, 646224 (2019), [Link](https://doi.org/doi:10.1101/646224).
16. M. Lizio et al., Update of the FANTOM web resource: Expansion to provide additional transcriptome atlases. __Nucleic Acids Res.__ 47, D752–D758 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1099).
17. S. Maedera et al., GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett.__ 593, 195–208 (2019), [Link](https://doi.org/doi:10.1002/1873-3468.13298).
18. T. Matsuda et al., Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__. 101, 472-485.e7 (2019), [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
19. Y. Murayama et al., Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett.__ 593, 423–432 (2019), [Link](https://doi.org/doi:10.1002/1873-3468.13328).
20. B. Oulès et al., Mutant Lef1 controls Gata6 in sebaceous gland development and cancer. __EMBO J.__ 38, e100526 (2019), [Link](https://doi.org/doi:10.15252/embj.2018100526).
21. C. Peggion et al., The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol. Neurobiol.__ 56, 3420–3436 (2019), [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
22. F.-C. Qian et al., SEanalysis: a web tool for super-enhancer associated regulatory analysis. __Nucleic Acids Res.__ (2019), [Link](https://doi.org/doi:10.1093/nar/gkz302).
23. M. Ranjit et al., Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep.__ 26, 2274-2281.e5 (2019), [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
24. M. Saito et al., A Polymorphic Variant in p19 Arf Confers Resistance to Chemically Induced Skin Tumors by Activating the p53 Pathway. __J. Invest. Dermatol.__ (2019), [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
25. R. Singh et al., Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat. Commun.__ 10, 89 (2019), [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
26. S. Sofiadis et al., HMGB1 as a rheostat of chromatin topology and RNA homeostasis on the path to senescence. __bioRxiv Genomics__, 1–33 (2019), [Link](https://doi.org/doi:10.1101/540146).
27. J. B. Studd et al., Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__. 33, 1–14 (2019), [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
28. T. Sugasawa, T. Yoshiya, Y. Takeuchi, N. Yahagi, R. Sharma, Influence of intermittent cold stimulations on CREB and its targeting genes in muscle : investigations into molecular mechanisms of local cryotherapy. __Preprints__, 1–21 (2019), [Link](https://doi.org/doi:10.20944/preprints201904.0277.v1).
29. A. Tano et al., The juvenility-associated long noncoding RNA Gm14230 maintains cellular juvenescence . __J. Cell Sci.__ 132, jcs227801 (2019), [Link](https://doi.org/doi:10.1242/jcs.227801).
30. Z. Tong, Q. Cui, J. Wang, Y. Zhou, TransmiR v2.0: An updated transcription factor-microRNA regulation database. __Nucleic Acids Res.__ 47, D253–D258 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1023).
31. M. T. Ueda et al., Comprehensive genomic analysis reveals dynamic evolution of mammalian transposable elements that code for viral-like protein domains. __bioRxiv__, 628875 (2019), [Link](https://doi.org/doi:10.1101/628875).
32. W. J. Verdenius, The Genetic Basis of Transcriptional and Spatial Heterogeneity of Squamous Features in Pancreatic Ductal Adenocarcinoma. __bioRxiv__, 548354 (2019), [Link](https://doi.org/doi:10.1101/548354).
33. T. Yamada, N. Akimitsu, Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __Wiley Interdiscip. Rev. RNA__. 10, e1508 (2019), [Link](https://doi.org/doi:10.1002/wrna.1508).
34. I. Yevshin, R. Sharipov, S. Kolmykov, Y. Kondrakhin, F. Kolpakov, GTRD: A database on gene transcription regulation - 2019 update. __Nucleic Acids Res.__ 47, D100–D105 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1128).
35. K. Yoshizaki et al., Paternal Age Affects Offspring’S Behavior Possibly Via Epigenetic Mechanism Recruiting a Transcriptional Repressor Rest. __Eur. Neuropsychopharmacol.__ 29, S831 (2019), [Link](https://doi.org/doi:10.1016/j.euroneuro.2017.08.089).
36. K. Anan et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res.__ 46, 5441–5454 (2018), [Link](https://doi.org/doi:10.1093/nar/gky234).
37. S. Baldi et al., Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol. Cell__. 72, 661-672.e4 (2018), [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
38. S. Chatterjee et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res. Treat.__ 170, 431–438 (2018), [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
39. Y. Chen et al., PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci.__ 109, 3532–3542 (2018), [Link](https://doi.org/doi:10.1111/cas.13796).
40. J. Chèneby, M. Gheorghe, M. Artufel, A. Mathelier, B. Ballester, ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res.__ 46, D267–D275 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx1092).
41. T. Chishima, J. Iwakiri, M. Hamada, Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes (Basel).__ 9, 23 (2018), [Link](https://doi.org/doi:10.3390/genes9010023).
42. C. Cui et al., Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics, Proteomics Bioinforma.__ 16, 200–211 (2018), [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
43. R. Dréos, G. Ambrosini, R. Groux, R. C. Périer, P. Bucher, MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res.__ 46, D175–D180 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx995).
44. Y. Egashira, Y. Mori, Y. Yanagawa, S. Takamori, Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci. Rep.__ 8, 15156 (2018), [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
45. E. Ferris, L. M. Abegglen, J. D. Schiffman, C. Gregg, Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep.__ 22, 2742–2755 (2018), [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
46. P. Fiziev, J. Ernst, ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol.__ 19, 109 (2018), [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
47. H. Imrichova, S. Aerts, ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__, 388934 (2018), [Link](https://doi.org/doi:10.1101/388934).
48. H. Kawaji, dirHub: a trackHub configurator with directory structure projection. __bioRxiv__, 314807 (2018), [Link](https://doi.org/doi:10.1101/314807).
49. K. Kawata et al., Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__. 7, 212–229 (2018), [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
50. T. Kehl et al., REGGAE: A novel approach for the identification of key transcriptional regulators. __Bioinformatics__. 34, 3503–3510 (2018), [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
51. K. Khetchoumian et al., Cell-autonomous transcriptional mechanism for enhancement of translation capacity in secretory cells. __bioRxiv__, 454421 (2018), [Link](https://doi.org/doi:10.1101/454421).
52. M. Kikuchi, N. Hara, M. Hasegawa, A. Miyashita, R. Kuwano, Enhancer variants associated with Alzheimer ’ s disease affect gene expression via chromatin looping. __bioRxiv__, 426312 (2018), [Link](https://doi.org/doi:10.1101/426312).
53. C. A. Lareau et al., Supp Info: Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__, 255224 (2018), [Link](https://doi.org/doi:10.1101/255224).
54. F. Mao et al., EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res.__ 46, D92–D99 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx918).
55. A. Misawa, H. Orimo, lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif. Tissue Int.__ 103, 422–430 (2018), [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
56. I. Miura et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J. Psychiatr. Res.__ 100, 84–87 (2018), [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
57. K. Mochizuki et al., SETDB1 is essential for mouse primordial germ cell fate determination by ensuring BMP signaling. __Development__. 145, dev164160 (2018), [Link](https://doi.org/doi:10.1242/dev.164160).
58. R. Mourad, K. Ginalski, G. Legube, O. Cuvier, Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol.__ 19, 1–15 (2018), [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
59. Y. Nishizawa et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__. 9 (2018), [Link](https://doi.org/doi:10.18632/oncotarget.23554).
60. S. Ochsner et al., The Signaling Pathways Project: an integrated ’omics knowledgebase for mammalian cellular signaling pathways. __bioRxiv__, 401729 (2018), [Link](https://doi.org/doi:10.1101/401729).
61. Y. Onodera et al., Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__. 13, e0204860 (2018), [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
62. N. Polouliakh et al., Sequence homology in eukaryotes (SHOE): interactive visual tool for promoter analysis. __BMC Genomics__. 19, 715 (2018), [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
63. L. M. Raffield et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet.__ 14, e1007293 (2018), [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
64. S. L. Romero-Cordoba et al., Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci. Rep.__ 8, 12252 (2018), [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
65. A. Swoboda et al., STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pigmentation pathway. __bioRxiv__, 422832 (2018), [Link](https://doi.org/doi:10.1101/422832).
66. H. Wang et al., Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J. Biol. Chem.__ 293, 14740–14757 (2018), [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
67. Y. Chen, M. Widschwendter, A. E. Teschendorff, Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol.__ 18, 236 (2017), [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
68. O. Govaere et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__. 36, 6605–6616 (2017), [Link](https://doi.org/doi:10.1038/onc.2017.260).
69. M. H. Guo et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc. Natl. Acad. Sci.__ 114, E327–E336 (2017), [Link](https://doi.org/doi:10.1073/pnas.1619052114).
70. K. Ishigaki et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat. Genet.__ 49, 1120–1125 (2017), [Link](https://doi.org/doi:10.1038/ng.3885).
71. Z. Kalender Atak et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med.__ 9, 80 (2017), [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
72. T. Kehl et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res.__ 45, W146–W153 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx350).
73. K. Matsuda et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__. 144, 1948–1958 (2017), [Link](https://doi.org/doi:10.1242/dev.143479).
74. A. Naderi, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__. 8, 57907–57933 (2017), [Link](https://doi.org/doi:10.18632/oncotarget.17826).
75. Y. Onodera et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__. 16, 1369–1380 (2017), [Link](https://doi.org/doi:10.1111/acel.12680).
76. T. Sanosaka et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep.__ 20, 2992–3003 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
77. K. Tanegashima et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes to Cells__. 22, 71–83 (2017), [Link](https://doi.org/doi:10.1111/gtc.12456).
78. F. K. Turrell et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev.__ 31, 1339–1353 (2017), [Link](https://doi.org/doi:10.1101/gad.298463.117).
79. T. Umeyama, T. Ito, DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep.__ 21, 289–300 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
80. I. Yevshin, R. Sharipov, T. Valeev, A. Kel, F. Kolpakov, GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res.__ 45, D61–D67 (2017), [Link](https://doi.org/doi:10.1093/nar/gkw951).
81. M. Yoshihara et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep.__ 21, 308–315 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
82. N. Sugeno et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci. Rep.__ 6, 36328 (2016), [Link](https://doi.org/doi:10.1038/srep36328).

