## To cite ChIP-Atlas in your publication:

- Paper
  - Shinya Oki, Tazro Ohta, et al. ChIP‐Atlas: a data‐mining suite powered by full integration of public ChIP‐seq data. EMBO Rep. (2018) e46255; doi: https://doi.org/10.15252/embr.201846255
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

## Publications citing ChIP-Atlas:

1. A. Tano et al., J. Cell Sci., in press, [Link](https://doi.org/doi:10.1242/jcs.227801).
2. S. Sofiadis et al., HMGB1 as a rheostat of chromatin topology and RNA homeostasis on the path to senescence. __bioRxiv Genomics__, 540146 (2019), [Link](https://doi.org/doi:10.1101/540146).
3. W. J. Verdenius, The Genetic Basis of Transcriptional and Spatial Heterogeneity of Squamous Features in Pancreatic Ductal Adenocarcinoma. __bioRxiv__, 548354 (2019), [Link](https://doi.org/doi:10.1101/548354).
4. Z. Tong, Q. Cui, J. Wang, Y. Zhou, TransmiR v2.0: an updated transcription factor-microRNA regulation database. __Nucleic Acids Res.__ 47, D253–D258 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1023).
5. Y. Ishijima et al., The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Mol. Cell. Endocrinol.__ 483, 39–49 (2019), [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
6. K. Yoshizaki et al., Paternal age affects offspring’s behavior possibly via an epigenetic mechanism recruiting a transcriptional repressor REST. __bioRxiv__, 550095 (2019), [Link](https://doi.org/doi:10.1101/550095).
7. S. Chandrashekharappa et al., Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat. Commun.__ 10, 89 (2019), [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
8. T. Yamada, N. Akimitsu, Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __Wiley Interdiscip. Rev. RNA__. 10, e1508 (2019), [Link](https://doi.org/doi:10.1002/wrna.1508).
9. B. Oulès et al., EMBO J., in press, [Link](https://doi.org/doi:10.15252/embj.2018100526).
10. Y. Murayama et al., Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett.__ (2019), [Link](https://doi.org/doi:10.1002/1873-3468.13328).
11. T. Kehl et al., The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int. J. Cancer__. 144, 1432–1443 (2019), [Link](https://doi.org/doi:10.1002/ijc.31834).
12. M. Ranjit et al., Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep.__ 26, 2274–2281.e5 (2019), [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
13. C. Tanikawa et al., A polymorphic variant in p19Arf confers resistance to chemically-induced skin tumors by activating the p53 pathway. __J. Invest. Dermatol.__ (2019), [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
14. M. Lizio et al., Update of the FANTOM web resource: expansion to provide additional transcriptome atlases. __Nucleic Acids Res.__ 47, D752–D758 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1099).
15. J. B. Studd et al., Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__. 33, 1–14 (2019), [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
16. F. Miura et al., Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__ (2019), [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
17. I. Yevshin, R. Sharipov, S. Kolmykov, Y. Kondrakhin, F. Kolpakov, GTRD: a database on gene transcription regulation-2019 update. __Nucleic Acids Res.__ 47, D100–D105 (2019), [Link](https://doi.org/doi:10.1093/nar/gky1128).
18. Ö. Åkerborg et al., High-resolution Regulatory Maps Connect Vascular Risk Variants to Disease Related Pathways. __Circ. Genomic Precis. Med.__ (2019), [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
19. S. Banerjee et al., Identifying Transcriptional Regulatory Modules Among Different Chromatin States in Mouse Neural Stem Cells. __Front. Genet.__ 9, 731 (2019), [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
20. P. Fiziev, J. Ernst, ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol.__ 19, 109 (2018), [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
21. Y. Egashira, Y. Mori, Y. Yanagawa, S. Takamori, Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci. Rep.__ 8, 15156 (2018), [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
22. I. Miura et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J. Psychiatr. Res.__ 100, 84–87 (2018), [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
23. H. Kitano et al., Sequence homology in eukaryotes (SHOE): interactive visual tool for promoter analysis. __BMC Genomics__. 19, 715 (2018), [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
24. H. Kawaji, dirHub: a trackHub configurator with directory structure projection. __bioRxiv__, 314807 (2018), [Link](https://doi.org/doi:10.1101/314807).
25. Y. Onodera et al., Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__. 13, e0204860 (2018), [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
26. K. Khetchoumian et al., Cell-autonomous transcriptional mechanism for enhancement of translation capacity in secretory cells. __bioRxiv__, 454421 (2018), [Link](https://doi.org/doi:10.1101/454421).
27. T. Soga et al., GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett.__ 593, 195–208 (2018), [Link](https://doi.org/doi:10.1002/1873-3468.13298).
28. S. Chatterjee et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res. Treat.__ 170, 431–438 (2018), [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
29. H.-P. Lenhof et al., REGGAE: a novel approach for the identification of key transcriptional regulators. __Bioinformatics__. 34, 3503–3510 (2018), [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
30. S. Cagnin et al., The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol. Neurobiol.__ (2018), [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
31. C. A. Lareau et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__ (2018), [Link](https://doi.org/doi:10.1101/255224).
32. T. Sekinaka et al., SETDB1 is essential for mouse primordial germ cell fate determination by ensuring BMP signaling. __Development__. 145, dev164160 (2018), [Link](https://doi.org/doi:10.1242/dev.164160).
33. C. A. Lareau et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__, 255224 (2018), [Link](https://doi.org/doi:10.1101/255224).
34. M. Kikuchi et al., Enhancer variants associated with Alzheimer’s disease affect gene expression via chromatin looping. __bioRxiv__, 426312 (2018), [Link](https://doi.org/doi:10.1101/426312).
35. C. Cui et al., Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics, Proteomics Bioinforma.__ 16, 200–211 (2018), [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
36. K. Kohrogi et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res.__ 46, 5441–5454 (2018), [Link](https://doi.org/doi:10.1093/nar/gky234).
37. T. Chishima, J. Iwakiri, M. Hamada, Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes (Basel).__ 9, 23 (2018), [Link](https://doi.org/doi:10.3390/genes9010023).
38. S. Baldi et al., Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol. Cell__. 72, 661–672.e4 (2018), [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
39. S. Ochsner et al., The Signaling Pathways Project: an integrated ’omics knowledgebase for mammalian cellular signaling pathways. __bioRxiv__, 401729 (2018), [Link](https://doi.org/doi:10.1101/401729).
40. S. Rodriguez-Cuevas et al., Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci. Rep.__ 8, 12252 (2018), [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
41. K. Pietras et al., Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__. 22, 117–131 (2018), [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
42. A. Misawa, H. Orimo, lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif. Tissue Int.__ 103, 422–430 (2018), [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
43. R. Dréos, G. Ambrosini, R. Groux, R. C. Périer, P. Bucher, MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res.__ 46, D175–D180 (2018), [Link](https://doi.org/doi:10.1093/nar/gkx995).
44. E. Ferris, L. M. Abegglen, J. D. Schiffman, C. Gregg, Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep.__ 22, 2742–2755 (2018), [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
45. S. Uda et al., Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__. 7, 212–229 (2018), [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
46. L. M. Raffield et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet.__ 14, e1007293 (2018), [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
47. Y. Chen et al., PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci.__ 109, 3532–3542 (2018), [Link](https://doi.org/doi:10.1111/cas.13796).
48. R. Mourad, K. Ginalski, G. Legube, O. Cuvier, Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol.__ 19, 1–15 (2018), [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
49. H. Wang et al., Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J. Biol. Chem.__ 293, 14740–14757 (2018), [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
50. H. Imrichova, S. Aerts, ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__, 388934 (2018), [Link](https://doi.org/doi:10.1101/388934).
51. A. Swoboda et al., STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pigmentation pathway. __bioRxiv__, 422832 (2018), [Link](https://doi.org/doi:10.1101/422832).
52. B. Ballester, A. Mathelier, M. Artufel, J. Chèneby, M. Gheorghe, ReMap 2018: an updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res.__ 46, D267–D275 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx1092).
53. Y. Hayashizaki et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep.__ 21, 308–315 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
54. S. Aerts et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med.__ 9, 80 (2017), [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
55. O. Govaere et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__. 36, 6605–6616 (2017), [Link](https://doi.org/doi:10.1038/onc.2017.260).
56. A. Naderi, C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__. 8, 57907–57933 (2017), [Link](https://doi.org/doi:10.18632/oncotarget.17826).
57. K. Matsuda et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__. 144, 1948–1958 (2017), [Link](https://doi.org/doi:10.1242/dev.143479).
58. K. Ishigaki et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat. Genet.__ 49, 1120–1125 (2017), [Link](https://doi.org/doi:10.1038/ng.3885).
59. K. Fukuda et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__. 16, 1369–1380 (2017), [Link](https://doi.org/doi:10.1111/acel.12680).
60. T. Kudo et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__. 9 (2017), [Link](https://doi.org/doi:10.18632/oncotarget.23554).
61. F. K. Turrell et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev.__ 31, 1339–1353 (2017), [Link](https://doi.org/doi:10.1101/gad.298463.117).
62. T. Kehl et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res.__ 45, W146–W153 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx350).
63. T. Imamura et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep.__ 20, 2992–3003 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
64. Q. Liu et al., EpiDenovo: a platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res.__ 46, D92–D99 (2017), [Link](https://doi.org/doi:10.1093/nar/gkx918).
65. I. Yevshin, R. Sharipov, T. Valeev, A. Kel, F. Kolpakov, GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res.__ 45, D61–D67 (2017), [Link](https://doi.org/doi:10.1093/nar/gkw951).
66. Y. Chen, M. Widschwendter, A. E. Teschendorff, Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol.__ 18, 236 (2017), [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
67. T. Umeyama, T. Ito, DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep.__ 21, 289–300 (2017), [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
68. R. Chiarle et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc. Natl. Acad. Sci.__ 114, E327–E336 (2016), [Link](https://doi.org/doi:10.1073/pnas.1619052114).
69. N. Sugeno et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci. Rep.__ 6, 36328 (2016), [Link](https://doi.org/doi:10.1038/srep36328).
70. T. Aigaki et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice . __Genes to Cells__. 22, 71–83 (2016), [Link](https://doi.org/doi:10.1111/gtc.12456).
