## To cite ChIP-Atlas in your publication:

- Paper
  - Shinya Oki, Tazro Ohta, et al. ChIP‐Atlas: a data‐mining suite powered by full integration of public ChIP‐seq data. EMBO Rep. (2018) e46255; doi: https://doi.org/10.15252/embr.201846255
- Database
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://dx.doi.org/10.18908/lsdba.nbdc01558-000
- Web tool
  - Oki, S; Ohta, T (2015): ChIP-Atlas. http://chip-atlas.org

## Publications citing ChIP-Atlas:

1. AS, Knaupp, et al., TINC- A Method to Dissect Regulatory Complexes at Single-Locus Resolution- Reveals an Extensive Protein Complex at the Nanog Promoter. __Stem Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.STEMCR.2020.11.005).
2. Ahmed, Mahmoud, et al., Integrating binding and expression data to predict transcription factors combined function. __BMC Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12864-020-06977-1).
3. Beknazarov, Nazar, et al., Deep learning approach for predicting functional Z-DNA regions using omics data. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-76203-1).
4. Bono, Hidemasa, et al., Meta-analysis of hypoxic transcriptomes from public databases. __Biomedicines__ (2020). [Link](https://doi.org/doi:10.3390/biomedicines8010010).
5. Booth, Lauren N, et al., Males deploy multifaceted strategies and hijack longevity pathways to induce premature demise of the opposite sex. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.30.181008).
6. Bunina, Daria, et al., Genomic Rewiring of SOX2 Chromatin Interaction Network during Differentiation of ESCs to Postmitotic Neurons. __Cell Syst__ (2020). [Link](https://doi.org/doi:10.1016/j.cels.2020.05.003).
7. Caruana, Beth T., et al., The NF-κB signalling pathway regulates GLUT6 expression in endometrial cancer. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109688).
8. Cataldo, Alessandra, et al., MiR-302b as a Combinatorial Therapeutic Approach to Improve Cisplatin Chemotherapy Efficacy in Human Triple-Negative Breast Cancer. __Cancers__ (2020). [Link](https://doi.org/doi:10.3390/cancers12082261).
9. Cavalli, Marco, et al., A Multi-Omics Approach to Liver Diseases: Integration of Single Nuclei Transcriptomics with Proteomics and HiCap Bulk Data in Human Liver. __OMICS__ (2020). [Link](https://doi.org/doi:10.1089/omi.2019.0215).
10. Chauvistré, Heike, et al., The Janus-faced role of KDM5B heterogeneity in melanoma: differentiation as a situational driver of both growth arrest and drug-resistance. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.01.999847).
11. Chen, Carol C L, et al., Histone H3.3G34-Mutant Interneuron Progenitors Co-opt PDGFRA for Gliomagenesis. __Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.cell.2020.11.012).
12. Chen, Jiaxin, et al., LncSEA: a platform for long non-coding RNA related sets and enrichment analysis. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa806).
13. Choi, Jang Hyun, et al., Histone demethylase KDM4D cooperates with NFIB and MLL1 complex to regulate adipogenic differentiation of C3H10T1/2 mesenchymal stem cells. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-60049-8).
14. Choudhury, Abrar, et al., Meningioma epigenetic grouping reveals biologic drivers and therapeutic vulnerabilities. __medRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.11.23.20237495).
15. Cofer, Evan M., et al., DeepArk: modeling cis-regulatory codes of model species with deep learning. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.23.058040).
16. Czipa, Erik, et al., ChIPSummitDB: a ChIP-seq-based database of human transcription factor binding sites and the topological arrangements of the proteins bound to them. __Database__ (2020). [Link](https://doi.org/doi:10.1093/database/baz141).
17. Dhaka, Bhavya, et al., Differential chromatin accessibility landscape of gain-of-function mutant p53 tumours. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.11.24.395913).
18. Dorotea, Debra, et al., Recent Insights Into SREBP as a Direct Mediator of Kidney Fibrosis via Lipid-Independent Pathways. __Front Pharmacol__ (2020). [Link](https://doi.org/doi:10.3389/fphar.2020.00265).
19. Farahmand, Saman, et al., ModEx: A text mining system for extracting mode of regulation of transcription factor-gene regulatory interaction. __J Biomed Inform__ (2020). [Link](https://doi.org/doi:10.1016/j.jbi.2019.103353).
20. Feng, Chenchen, et al., KnockTF: A comprehensive human gene expression profile database with knockdown/knockout of transcription factors. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz881).
21. Fornes, Oriol, et al., JASPAR 2020: Update of the open-Access database of transcription factor binding profiles. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1001).
22. Gao, Lihua, et al., Ozone therapy promotes the differentiation of basal keratinocytes via increasing Tp63-mediated transcription of KRT10 to improve psoriasis. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15160).
23. Gerisch, Birgit, et al., HLH-30/TFEB Is a Master Regulator of Reproductive Quiescence. __Dev Cell__ (2020). [Link](https://doi.org/doi:10.1016/j.devcel.2020.03.014).
24. Guan, Shuhong, et al., TGF-β1 induces CREB1-mediated miR-1290 upregulation to antagonize lung fibrosis via Napsin A. __Int J Mol Med__ (2020). [Link](https://doi.org/doi:10.3892/ijmm.2020.4565).
25. Hayashi, Akimasa, et al., A unifying paradigm for transcriptional heterogeneity and squamous features in pancreatic ductal adenocarcinoma. __Nat Cancer__ (2020). [Link](https://doi.org/doi:10.1038/s43018-019-0010-1).
26. Hayashi, Akimasa, et al., Evolutionary Dynamics of Non-Coding Regions in Pancreatic Ductal Adenocarcinoma. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.11.294389).
27. Hirano, Kazumi, et al., Indonesian ginger (Bangle) extract promotes neurogenesis of human neural stem cells through WNT pathway activation. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134772).
28. Horn, Moritz, et al., Hexosamine Pathway Activation Improves Protein Homeostasis through the Integrated Stress Response. __iScience__ (2020). [Link](https://doi.org/doi:10.1016/j.isci.2020.100887).
29. Hsiao, Wen-Yu, et al., The Lipid Handling Capacity of Subcutaneous Fat Is Programmed by mTORC2 during Development. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/j.celrep.2020.108223).
30. Huang, Chia-Chi Flora, et al., Functional mapping of androgen receptor enhancer activity. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.08.18.255232).
31. Hänsel-Hertsch, Robert, et al., Landscape of G-quadruplex DNA structural regions in breast cancer. __Nat Genet__ (2020). [Link](https://doi.org/doi:10.1038/s41588-020-0672-8).
32. Jiang, Jiayue-Clara, et al., L1PA2 transposons contribute abundant regulatory sequences in MCF7 breast cancer cell line. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.01.276808).
33. Kaminuma, Eli, et al., DDBJ data analysis challenge: A machine learning competition to predict arabidopsis chromatin feature annotations from DNA sequences. __Genes Genet Syst__ (2020). [Link](https://doi.org/doi:10.1266/ggs.19-00034).
34. Katahira, Jun, et al., Human THO maintains the stability of repetitive DNA. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12760).
35. Kikuyama, Takahiro, et al., 25(OH)D3 stimulates the expression of vitamin D target genes in renal tubular cells when Cyp27b1 is abrogated. __J Steroid Biochem Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jsbmb.2020.105593).
36. Kim, Yu Jin, et al., UBE2C Overexpression Aggravates Patient Outcome by Promoting Estrogen-Dependent/Independent Cell Proliferation in Early Hormone Receptor-Positive and HER2-Negative Breast Cancer. __Front Oncol__ (2020). [Link](https://doi.org/doi:10.3389/fonc.2019.01574).
37. Knaupp, AS, et al., TINC - a method to dissect transcriptional complexes at single locus resolution - reveals novel Nanog regulators in mouse embryonic stem cells. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.03.023200).
38. Koga, Tomoaki, et al., Expression of leukotriene B4 receptor 1 defines functionally distinct DCs that control allergic skin inflammation. __Cell Mol Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41423-020-00559-7).
39. Kohmoto, Tomohiro, et al., Claudin-6 is a single prognostic marker and functions as a tumor-promoting gene in a subgroup of intestinal type gastric cancer. __Gastric Cancer__ (2020). [Link](https://doi.org/doi:10.1007/s10120-019-01014-x).
40. Kolmykov, Semyon, et al., GTRD: an integrated view of transcription regulation. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa1057).
41. Kremsky, Isaac, et al., Protection from DNA re-methylation by transcription factors in primordial germ cells and pre-implantation embryos can explain trans-generational epigenetic inheritance. __Genome Biol__ (2020). [Link](https://doi.org/doi:10.1186/s13059-020-02036-w).
42. Krześniak, Małgorzata, et al., Synergistic activation of p53 by actinomycin D and nutlin-3a is associated with the upregulation of crucial regulators and effectors of innate immunity. __Cell Signal__ (2020). [Link](https://doi.org/doi:10.1016/j.cellsig.2020.109552).
43. Kubota, Naoto, et al., An integrated analysis of public genomic data unveils a possible functional mechanism of psoriasis risk via a long-range ERRFI1 enhancer. __BMC Med Genomics__ (2020). [Link](https://doi.org/doi:10.1186/s12920-020-0662-9).
44. Kutscher, Lena, et al., Functional loss of a non-canonical BCOR-PRC1.1 complex accelerates SHH-driven medulloblastoma formation. __Genes Dev__ (2020). [Link](https://doi.org/doi:10.1101/2020.02.06.938035).
45. Kwiecien, Kamila, et al., The methylation status of the chemerin promoter region located from − 252 to + 258 bp regulates constitutive but not acute-phase cytokine-inducible chemerin expression levels. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-70625-7).
46. Kwon, Andrew Tae-Jun, et al., Efficient Development of Platform Cell Lines Using CRISPR-Cas9 and Transcriptomics Analysis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.09.16.299248).
47. L, He, et al., A Regulation Loop between YAP and NR4A1 Balances Cell Proliferation and Apoptosis. __Cell Rep__ (2020). [Link](https://doi.org/doi:10.1016/J.CELREP.2020.108284).
48. LM, Koch, et al., Cytosolic pH regulates proliferation and tumour growth by promoting expression of cyclin D1. __Nat Metab__ (2020). [Link](https://doi.org/doi:10.1038/S42255-020-00297-0).
49. Lei, Xiao, et al., Crystal Structures of Ternary Complexes of MEF2 and NKX2–5 Bound to DNA Reveal a Disease Related Protein–Protein Interaction Interface. __J Mol Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.jmb.2020.07.004).
50. Leporcq, Clémentine, et al., TFmotifView: a webserver for the visualization of transcription factor motifs in genomic regions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkaa252).
51. Li, Xin, et al., ERRα activates SHMT2 transcription to enhance the resistance of breast cancer to lapatinib via modulating the mitochondrial metabolic adaption. __Biosci Rep__ (2020). [Link](https://doi.org/doi:10.1042/BSR20192465).
52. Li, Yanyu, et al., TRlnc: a comprehensive database for human transcriptional regulatory information of lncRNAs. __Brief Bioinform__ (2020). [Link](https://doi.org/doi:10.1093/bib/bbaa011).
53. Li, Yi-Han, et al., FGF9 is a downstream target of SRY and sufficient to determine male sex fate in ex vivo XX gonad culture. __Biol Reprod__ (2020). [Link](https://doi.org/doi:10.1093/biolre/ioaa154).
54. Lin, Quy Xiao Xuan, et al., TFregulomeR reveals transcription factors' context-specific features and functions. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/nar/gkz1088).
55. Lyu, Junfang, et al., Synthetic lethality of RB1 and aurora A is driven by stathmin-mediated disruption of microtubule dynamics. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18872-0).
56. Maharjan, Mukesh, et al., Overlapping but Distinct Sequences Play Roles in the Insulator and Promoter Activities of the Drosophila BEAF-Dependent scs' Insulator. __BMC Genet__ (2020). [Link](https://doi.org/doi:10.1534/genetics.120.303344).
57. Mao, Ninghui, et al., Oncogenic ERG represses PI3K signaling through downregulation of IRS2. __Cancer Res__ (2020). [Link](https://doi.org/doi:10.1158/0008-5472.CAN-19-1394).
58. McErlean, Peter, et al., Epigenetic alterations underlie airway macrophage differentiation and phenotype during lung fibrosis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.12.04.410191).
59. Meng, Yuhong, et al., Transcriptomic landscape profiling of metformin-treated healthy mice: Implication for potential hypertension risk when prophylactically used. __J Cell Mol Med__ (2020). [Link](https://doi.org/doi:10.1111/jcmm.15472).
60. Mercatelli, Daniele, et al., Gene regulatory network inference resources: A practical overview. __Biochim Biophys Acta Gene Regul Mech__ (2020). [Link](https://doi.org/doi:10.1016/j.bbagrm.2019.194430).
61. Merrett, James E., et al., Identification of DNA response elements regulating expression of CCAAT/enhancer-binding protein (C/EBP) β and δ during early adipogenesis. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.05.22.110114).
62. Miyamoto, Yoichi, et al., Genetic loss of importin α4 causes abnormal sperm morphology and impacts on male fertility in mouse. __FASEB J__ (2020). [Link](https://doi.org/doi:10.1096/fj.202000768RR).
63. Nam, A. Reum, et al., Alternative methylation of intron motifs is associated with cancer-related gene expression in both canine mammary tumor and human breast cancer. __Clin Epigenetics__ (2020). [Link](https://doi.org/doi:10.1186/s13148-020-00888-4).
64. Namba, Shinichi, et al., Multi-sample Full-length Transcriptome Analysis of 22 Breast Cancer Clinical Specimens with Long-Read Sequencing. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.07.15.199851).
65. Namekawa, Takeshi, et al., ALDH1A1 in patient-derived bladder cancer spheroids activates retinoic acid signaling leading to TUBB3 overexpression and tumor progression. __Int J Cancer__ (2020). [Link](https://doi.org/doi:10.1002/ijc.32505).
66. Namekawa, Takeshi, et al., HIF1α inhibitor 2-methoxyestradiol decreases NRN1 expression and represses in vivo and in vitro growth of patient-derived testicular germ cell tumor spheroids. __Cancer Lett__ (2020). [Link](https://doi.org/doi:10.1016/j.canlet.2020.05.040).
67. Ochsner, Scott A, et al., A transcriptional regulatory atlas of coronavirus infection of human cells. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.24.059527).
68. Ohishi, Hiroaki, et al., Characterization of genetic-origin-dependent monoallelic expression in mouse embryonic stem cells. __Genes Cells__ (2020). [Link](https://doi.org/doi:10.1111/gtc.12736).
69. Okada, Ryo, et al., RNaseH2A Downregulation Drives Chromosomal DNA Fragmentation and Accumulation of RNA-DNA Hybrids in Senescent Cells. __SSRN__ (2020). [Link](https://doi.org/doi:10.2139/ssrn.3588831).
70. Okazaki, Yuri, et al., Increased glucose metabolism in Arid5b-/- skeletal muscle is associated with the down-regulation of TBC1 domain family member 1 (TBC1D1). __Biol Res__ (2020). [Link](https://doi.org/doi:10.1186/s40659-020-00313-3).
71. P, Sahlén, et al., Chromatin interactions in differentiating keratinocytes reveal novel atopic dermatitis- and psoriasis-associated genes. __J Allergy Clin Immunol__ (2020). [Link](https://doi.org/doi:10.1016/J.JACI.2020.09.035).
72. Pecori, Federico, et al., Mucin-type O-glycosylation controls pluripotency in mouse embryonic stem cells via Wnt receptor endocytosis. __J Cell Sci__ (2020). [Link](https://doi.org/doi:10.1242/jcs.245845).
73. Pradhananga, Sailendra, et al., Promoter anchored interaction landscape of THP-1 macrophages captures early immune response processes. __Cell Immunol__ (2020). [Link](https://doi.org/doi:10.1016/j.cellimm.2020.104148).
74. Pratapa, Aditya, et al., Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data. __Nat Methods__ (2020). [Link](https://doi.org/doi:10.1038/s41592-019-0690-6).
75. Q, Pan, et al., VARAdb: a comprehensive variation annotation database for human. __Nucleic Acids Res__ (2020). [Link](https://doi.org/doi:10.1093/NAR/GKAA922).
76. Quinn, Hazel, et al., YAP and β-catenin co-operate to drive oncogenesis in basal breast cancer. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.05.115881).
77. Ray, John P., et al., Prioritizing disease and trait causal variants at the TNFAIP3 locus using functional and genomic features. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-15022-4).
78. Rikitake, Marika, et al., Analysis of GPI-anchored proteins involved in germline stem cell proliferation in the Caenorhabditis elegans germline stem cell niche. __J Biochem__ (2020). [Link](https://doi.org/doi:10.1093/jb/mvaa075).
79. Roels, Juliette, et al., Aging of preleukemic thymocytes drives CpG island hypermethylation in T-cell acute lymphoblastic leukemia. __Blood Cancer Discov__ (2020). [Link](https://doi.org/doi:10.1158/2643-3230.BCD-20-0059).
80. Roels, Juliette, et al., Distinct and temporary-restricted epigenetic mechanisms regulate human αβ and γδ T cell development. __Nat Immunol__ (2020). [Link](https://doi.org/doi:10.1038/s41590-020-0747-9).
81. Ronzio, Mirko, et al., Integrating Peak Colocalization and Motif Enrichment Analysis for the Discovery of Genome-Wide Regulatory Modules and Transcription Factor Recruitment Rules. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00072).
82. Ruiz, José L, et al., The regulatory genome of the malaria vector Anopheles gambiae: integrating 1 chromatin accessibility and gene expression 2. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.06.22.164228).
83. SN, Reisinger, et al., STAT3 in the dorsal raphe gates behavioural reactivity and regulates gene networks associated with psychopathology. __Mol Psychiatry__ (2020). [Link](https://doi.org/doi:10.1038/S41380-020-00904-2).
84. Sakuratani, Takuji, et al., Downregulation of ARID1A in gastric cancer cells: a putative protective molecular mechanism against the Harakiri-mediated apoptosis pathway. __Virchows Arch__ (2020). [Link](https://doi.org/doi:10.1007/s00428-020-02899-1).
85. Shikov, Anton E., et al., Phenome-wide functional dissection of pleiotropic effects highlights key molecular pathways for human complex traits. __Sci Rep__ (2020). [Link](https://doi.org/doi:10.1038/s41598-020-58040-4).
86. Sugasawa, Takehito, et al., Influence of intermittent cold stimulations on CREB and its targeting genes in muscle: Investigations into molecular mechanisms of local cryotherapy. __Int J Mol Sci__ (2020). [Link](https://doi.org/doi:10.3390/ijms21134588).
87. Sukumaran, Abitha, et al., Insight on Transcriptional Regulation of the Energy Sensing AMPK and Biosynthetic mTOR Pathway Genes. __Front Cell Dev Biol__ (2020). [Link](https://doi.org/doi:10.3389/fcell.2020.00671).
88. T, Kokaji, et al., Transomics analysis reveals allosteric and gene regulation axes for altered hepatic glucose-responsive metabolism in obesity. __Sci Signal__ (2020). [Link](https://doi.org/doi:10.1126/SCISIGNAL.AAZ1236).
89. Tamura, Isao, et al., Transcription factor C/EBPβ induces genome-wide H3K27ac and upregulates gene expression during decidualization of human endometrial stromal cells. __Molec Cell Endocr__ (2020). [Link](https://doi.org/doi:10.1016/j.mce.2020.111085).
90. Tanaka, Hiroshi, et al., The NSD2/WHSC1/MMSET methyltransferase prevents cellular senescence-associated epigenomic remodeling. __Aging Cell__ (2020). [Link](https://doi.org/doi:10.1111/acel.13173).
91. Teschendorff, Andrew E, et al., Improved detection of tumor suppressor events in 1 single-cell RNA-Seq data 2 3. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.07.04.187781).
92. Totani, Haruhito, et al., Autocrine HGF/c-Met signaling pathway confers aggressiveness in lymph node adult T-cell leukemia/lymphoma. __Oncogene__ (2020). [Link](https://doi.org/doi:10.1038/s41388-020-01393-x).
93. Wang, Allen, et al., Single Nucleus Multiomic Profiling Reveals Age-Dynamic Regulation of Host Genes Associated with SARS-CoV-2 Infection. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.04.12.037580).
94. Wang, Rong, et al., Transient IGF-1R inhibition combined with osimertinib eradicates AXL-low expressing EGFR mutated lung cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18442-4).
95. Wang, Zhenjia, et al., BART3D: Inferring transcriptional regulators associated with differential chromatin interactions from Hi-C data. __bioRxiv__ (2020). [Link](https://doi.org/doi:10.1101/2020.08.19.258095).
96. Wu, Xue, et al., Regulation of cellular sterol homeostasis by the oxygen responsive noncoding RNA lincNORS. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-18411-x).
97. X, Li, et al., The role of MYB proto-oncogene like 2 in tamoxifen resistance in breast cancer. __J Mol Histol__ (2020). [Link](https://doi.org/doi:10.1007/S10735-020-09920-6).
98. Y, Li, et al., Single-Cell Transcriptome Analysis Reveals Dynamic Cell Populations and Differential Gene Expression Patterns in Control and Aneurysmal Human Aortic Tissue. __Circulation__ (2020). [Link](https://doi.org/doi:10.1161/CIRCULATIONAHA.120.046528).
99. Y, Sera, et al., UTX maintains functional integrity of murine hematopoietic system by globally regulating aging-associated genes. __Blood__ (2020). [Link](https://doi.org/doi:10.1182/BLOOD.2019001044).
100. Y, Yasukawa, et al., SAA1 is upregulated in gastric cancer-associated fibroblasts possibly by its enhancer activation. __Carcinogenesis__ (2020). [Link](https://doi.org/doi:10.1093/CARCIN/BGAA131).
101. Yumimoto, Kanae, et al., Recent insight into the role of FBXW7 as a tumor suppressor. __Semin Cancer Biol__ (2020). [Link](https://doi.org/doi:10.1016/j.semcancer.2020.02.017).
102. Zhao, Wei, et al., Epigenetic Regulation of m6A Modifications in Human Cancer. __Mol Ther Nucleic Acids__ (2020). [Link](https://doi.org/doi:10.1016/j.omtn.2019.11.022).
103. Zhou, Stanley, et al., Noncoding mutations target cis-regulatory elements of the FOXA1 plexus in prostate cancer. __Nat Commun__ (2020). [Link](https://doi.org/doi:10.1038/s41467-020-14318-9).
104. Zhou, Yao, et al., epiCOLOC: Integrating Large-Scale and Context-Dependent Epigenomics Features for Comprehensive Colocalization Analysis. __Front Genet__ (2020). [Link](https://doi.org/doi:10.3389/fgene.2020.00053).
105. Abugessaisa, Imad, et al., refTSS: A Reference Data Set for Human and Mouse Transcription Start Sites. __J Mol Biol__ (2019). [Link](https://doi.org/doi:10.1016/j.jmb.2019.04.045).
106. Ahn, Hongryul, et al., Propanet: Time-varying condition-specific transcriptional network construction by network propagation. __Front Plant Sci__ (2019). [Link](https://doi.org/doi:10.3389/fpls.2019.00698).
107. Ando, Mizuo, et al., Chromatin dysregulation and DNA methylation at transcription start sites associated with transcriptional repression in cancers. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-09937-w).
108. Aono, Shiori, et al., β-catenin/TCF4 complex-mediated induction of the NRF3 (NFE2L3) gene in cancer cells. __Int J Mol Sci__ (2019). [Link](https://doi.org/doi:10.3390/ijms20133344).
109. Arata, Yoshiyuki, et al., Defective induction of the proteasome associated with T-cell receptor signaling underlies T-cell senescence. __Genes Cells__ (2019). [Link](https://doi.org/doi:10.1111/gtc.12728).
110. Bakhmet, Evgeny I., et al., hnRNP-K Targets Open Chromatin in Mouse Embryonic Stem Cells in Concert with Multiple Regulators. __Stem Cells__ (2019). [Link](https://doi.org/doi:10.1002/stem.3025).
111. Banerjee, Sharmi, et al., Identifying transcriptional regulatory modules among different chromatin states in mouse neural stem cells. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2018.00731).
112. Bocci, Matteo, et al., Activin receptor-like kinase 1 is associated with immune cell infiltration and regulates CLEC14A transcription in cancer. __Angiogenesis__ (2019). [Link](https://doi.org/doi:10.1007/s10456-018-9642-5).
113. Bravo González-Blas, Carmen, et al., cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. __Nat Methods__ (2019). [Link](https://doi.org/doi:10.1038/s41592-019-0367-1).
114. Chang, Hao Chun, et al., Investigating the role of super-enhancer RNAs underlying embryonic stem cell differentiation. __BMC Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12864-019-6293-x).
115. Chen, Dijun, et al., ChIP-Hub: an Integrative Platform for Exploring Plant Regulome. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/768903).
116. Chereji, Răzvan V., et al., Quantitative MNase-seq accurately maps nucleosome occupancy levels. __Genome Biol__ (2019). [Link](https://doi.org/doi:10.1186/s13059-019-1815-z).
117. Delicado, Diana, et al., Multilocus phylogeny, species delimitation and biogeography of Iberian valvatiform springsnails (Caenogastropoda: Hydrobiidae), with the description of a new genus. __Zool J Linn Soc__ (2019). [Link](https://doi.org/doi:10.1093/zoolinnean/zly093).
118. Duforestel, Manon, et al., Glyphosate primes mammary cells for tumorigenesis by reprogramming the epigenome in a TET3-dependent manner. __Front Genet__ (2019). [Link](https://doi.org/doi:10.3389/fgene.2019.00885).
119. Farahmand, Saman, et al., Causal Inference Engine: A platform for directional gene set enrichment analysis and inference of active transcriptional regulators. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz1046).
120. Ferris, Elliott, et al., Parallel Accelerated Evolution in Distant Hibernators Reveals Candidate Cis Elements and Genetic Circuits Regulating Mammalian Obesity. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.10.102).
121. Fontela, Miguel G., et al., The conserved non-coding sequence 2 (CNS2) enhances CD69 transcription through cooperation between the transcription factors oct1 and RUNX1. __Genes__ (2019). [Link](https://doi.org/doi:10.3390/genes10090651).
122. Grujic, Olivera, et al., Identification and characterization of constrained non-exonic bases lacking predictive epigenomic and transcription factor binding annotations. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/722876).
123. Hayashi, Akimasa, et al., The Genetic Basis of Transcriptional and Spatial Heterogeneity of Squamous Features in Pancreatic Ductal Adenocarcinoma. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/548354).
124. Horie, Kenta, et al., Down-regulation of GATA1-dependent erythrocyte-related genes in the spleens of mice exposed to a space travel. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-44067-9).
125. Hotta, Masahiro, et al., Farnesoid X receptor induces cell death and sensitizes to TRAIL-induced inhibition of growth in colorectal cancer cells through the up-regulation of death receptor 5. __Biochem Biophys Res Commun__ (2019). [Link](https://doi.org/doi:10.1016/j.bbrc.2019.09.033).
126. Hurtz, Christian, et al., Rationale for targeting BCL6 in MLL-rearranged acute lymphoblastic leukemia. __Genes Dev__ (2019). [Link](https://doi.org/doi:10.1101/gad.327593.119).
127. Ishijima, Yasushi, et al., The Gata2 repression during 3T3-L1 preadipocyte differentiation is dependent on a rapid decrease in histone acetylation in response to glucocorticoid receptor activation. __Molec Cell Endocr__ (2019). [Link](https://doi.org/doi:10.1016/j.mce.2019.01.002).
128. Kang, Hyeji, et al., HSF1 regulates mevalonate and cholesterol biosynthesis pathways. __Cancers__ (2019). [Link](https://doi.org/doi:10.3390/cancers11091363).
129. Kehl, Tim, et al., The role of TCF3 as potential master regulator in blastemal Wilms tumors. __Int J Cancer__ (2019). [Link](https://doi.org/doi:10.1002/ijc.31834).
130. Kikuchi, Masataka, et al., Enhancer variants associated with Alzheimer's disease affect gene expression via chromatin looping. __BMC Med Genomics__ (2019). [Link](https://doi.org/doi:10.1186/s12920-019-0574-8).
131. Klein, Jason C., et al., Functional testing of thousands of osteoarthritis-associated variants for regulatory activity. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-10439-y).
132. Kojima, Machiko, et al., The histone methyltransferase WHSC1 is regulated by EZH2 and is important for ovarian clear cell carcinoma cell proliferation. __BMC Cancer__ (2019). [Link](https://doi.org/doi:10.1186/s12885-019-5638-9).
133. Kokaji, Toshiya, et al., Trans-omic analysis reveals allosteric and gene regulation-axes for altered glucose-responsive liver metabolism associated with obesity. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/653758).
134. Kolmykov, Semyon K., et al., Population size estimation for quality control of ChIP-Seq datasets. __PLoS One__ (2019). [Link](https://doi.org/doi:10.1371/journal.pone.0221760).
135. Kondo, Takefumi, et al., Two-step regulation of trachealess ensures tight coupling of cell fate with morphogenesis in the drosophila trachea. __Elife__ (2019). [Link](https://doi.org/doi:10.7554/eLife.45145).
136. Krug, Brian, et al., Pervasive H3K27 Acetylation Leads to ERV Expression and a Therapeutic Vulnerability in H3K27M Gliomas. __Cancer Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.ccell.2019.04.004).
137. Kulyté, Agné, et al., MicroRNA-27a/b-3p and PPARG regulate SCAMP3 through a feed-forward loop during adipogenesis. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-50210-3).
138. Kunii, Yasuto, et al., Differential protein expression of DARPP-32 versus Calcineurin in the prefrontal cortex and nucleus accumbens in schizophrenia and bipolar disorder. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-51456-7).
139. Labade, Ajay, et al., Nup93 and CTCF co-modulate spatiotemporal dynamics and function of the HOXA gene cluster during differentiation. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/646224).
140. Lin, Siyuan, et al., Transcriptome and DNA methylome signatures associated with retinal Müller glia development, injury response, and aging. __Invest Ophthalmol Vis Sci__ (2019). [Link](https://doi.org/doi:10.1167/iovs.19-27361).
141. Lizio, Marina, et al., Update of the FANTOM web resource: Expansion to provide additional transcriptome atlases. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1099).
142. Lou, Shaoke, et al., TopicNet: a framework for measuring transcriptional regulatory network change. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/862177).
143. Maedera, Shotaro, et al., GLUT6 is a lysosomal transporter that is regulated by inflammatory stimuli and modulates glycolysis in macrophages. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13298).
144. Matsuda, Taito, et al., Pioneer Factor NeuroD1 Rearranges Transcriptional and Epigenetic Profiles to Execute Microglia-Neuron Conversion. __Neuron__ (2019). [Link](https://doi.org/doi:10.1016/j.neuron.2018.12.010).
145. Menzel, Michael, et al., Enhort: A platform for deep analysis of genomic positions. __PeerJ Comput Sci__ (2019). [Link](https://doi.org/doi:10.7717/peerj-cs.198).
146. Murayama, Yuki, et al., Glucocorticoid receptor suppresses gene expression of Rev-erbα (Nr1d1) through interaction with the CLOCK complex. __FEBS Lett__ (2019). [Link](https://doi.org/doi:10.1002/1873-3468.13328).
147. Neiman, Gabriel, et al., Integrin alpha-5 subunit is critical for the early stages of human pluripotent stem cell cardiac differentiation. __Sci Rep__ (2019). [Link](https://doi.org/doi:10.1038/s41598-019-54352-2).
148. Oulès, Bénédicte, et al., Mutant Lef1 controls Gata6 in sebaceous gland development and cancer. __EMBO J__ (2019). [Link](https://doi.org/doi:10.15252/embj.2018100526).
149. Peggion, Caterina, et al., The Prion Protein Regulates Synaptic Transmission by Controlling the Expression of Proteins Key to Synaptic Vesicle Recycling and Exocytosis. __Mol Neurobiol__ (2019). [Link](https://doi.org/doi:10.1007/s12035-018-1293-4).
150. Qian, Feng Cui, et al., SEanalysis: A web tool for super-enhancer associated regulatory analysis. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gkz302).
151. Ranjit, Melissa, et al., Aberrant Active cis-Regulatory Elements Associated with Downregulation of RET Finger Protein Overcome Chemoresistance in Glioblastoma. __Cell Rep__ (2019). [Link](https://doi.org/doi:10.1016/j.celrep.2019.01.109).
152. Reimegård, Johan, et al., Combined mRNA and protein single cell analysis in a dynamic cellular system using SPARC. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/749473).
153. Robinson, Claire M., et al., AHypoxia-inducible HIF1-GAL3ST1-sulfatide axis enhances ccRCC immune evasion via increased tumor cell-platelet binding. __Mol Cancer Res__ (2019). [Link](https://doi.org/doi:10.1158/1541-7786.MCR-19-0461).
154. Saito, Megumi, et al., A Polymorphic Variant in p19Arf Confers Resistance to Chemically Induced Skin Tumors by Activating the p53 Pathway. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2018.12.027).
155. Salameti, Vasiliki, et al., NOTCH1 signaling in oral squamous cell carcinoma via a TEL2/ SERPINE1 axis. __Oncotarget__ (2019). [Link](https://doi.org/doi:10.18632/oncotarget.27306).
156. Shikov, Anton, et al., Phenome-wide search for pleiotropic loci highlights key genes and molecular pathways for human complex traits. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/672758).
157. Simak, Maria, et al., Boolean function network analysis of time course liver transcriptome data to reveal novel circadian transcriptional regulators in mammals. __J Chin Med Assoc__ (2019). [Link](https://doi.org/doi:10.1097/JCMA.0000000000000180).
158. Singh, Rajbir, et al., Enhancement of the gut barrier integrity by a microbial metabolite through the Nrf2 pathway. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-018-07859-7).
159. Sofiadis, Konstantinos, et al., HMGB1 coordinates SASP-related chromatin folding and RNA homeostasis on the path to senescence. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/540146).
160. Solari, Claudia, et al., The pluripotency transcription factor Nanog represses glutathione reductase gene expression in mouse embryonic stem cells. __BMC Res Notes__ (2019). [Link](https://doi.org/doi:10.1186/s13104-019-4411-0).
161. Studd, James B., et al., Genetic predisposition to B-cell acute lymphoblastic leukemia at 14q11.2 is mediated by a CEBPE promoter polymorphism. __Leukemia__ (2019). [Link](https://doi.org/doi:10.1038/s41375-018-0184-z).
162. Suda, Kojiro, et al., Reduction of Rpd3 suppresses defects in locomotive ability and neuronal morphology induced by the knockdown of Drosophila SLC25A46 via an epigenetic pathway. __Exp Cell Res__ (2019). [Link](https://doi.org/doi:10.1016/j.yexcr.2019.111673).
163. Tano, Ayami, et al., The juvenility-associated long noncoding RNA Gm14230 maintains cellular juvenescence. __J Cell Sci__ (2019). [Link](https://doi.org/doi:10.1242/jcs.227801).
164. Tong, Zhan, et al., TransmiR v2.0: An updated transcription factor-microRNA regulation database. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1023).
165. Ueda, Mahoko Takahashi, et al., Comprehensive genomic analysis reveals dynamic evolution of endogenous retroviruses that code for retroviral-like protein domains. __bioRxiv__ (2019). [Link](https://doi.org/doi:10.1101/628875).
166. Velychko, Sergiy, et al., Excluding Oct4 from Yamanaka Cocktail Unleashes the Developmental Potential of iPSCs. __Cell Stem Cell__ (2019). [Link](https://doi.org/doi:10.1016/j.stem.2019.10.002).
167. Vijayakrishnan, Jayaram, et al., Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. __Nat Commun__ (2019). [Link](https://doi.org/doi:10.1038/s41467-019-13069-6).
168. Vogel, Felix C.E., et al., Targeting the H3K4 Demethylase KDM5B Reprograms the Metabolome and Phenotype of Melanoma Cells. __J Invest Dermatol__ (2019). [Link](https://doi.org/doi:10.1016/j.jid.2019.06.124).
169. Wu, Cheng Jang, et al., MiR-23~27~24–mediated control of humoral immunity reveals a TOX-driven regulatory circuit in follicular helper T cell differentiation. __Sci Adv__ (2019). [Link](https://doi.org/doi:10.1126/sciadv.aaw1715).
170. Wu, Zhaoming, et al., Cis-control of Six1 expression in neural crest cells during craniofacial development. __Dev Dyn__ (2019). [Link](https://doi.org/doi:10.1002/dvdy.109).
171. Yamada, Toshimichi, et al., Contributions of regulated transcription and mRNA decay to the dynamics of gene expression. __WIREs RNA__ (2019). [Link](https://doi.org/doi:10.1002/wrna.1508).
172. Yashiro, Takuya, et al., PU.1 plays a pivotal role in dendritic cell migration from the periphery to secondary lymphoid organs via regulating CCR7 expression. __FASEB J__ (2019). [Link](https://doi.org/doi:10.1096/fj.201900379RR).
173. Yevshin, Ivan, et al., GTRD: A database on gene transcription regulation - 2019 update. __Nucleic Acids Res__ (2019). [Link](https://doi.org/doi:10.1093/nar/gky1128).
174. Yoshizaki, Kaichi, et al., Paternal Age Affects Offspring'S Behavior Possibly Via Epigenetic Mechanism Recruiting a Transcriptional Repressor Rest. __Eur Neuropsychopharmacol__ (2019). [Link](https://doi.org/doi:10.1016/j.euroneuro.2017.08.089).
175. Åkerborg, Örjan, et al., High-Resolution Regulatory Maps Connect Vascular Risk Variants to Disease-Related Pathways. __Circulation__ (2019). [Link](https://doi.org/doi:10.1161/CIRCGEN.118.002353).
176. Anan, Kotaro, et al., LSD1 mediates metabolic reprogramming by glucocorticoids during myogenic differentiation. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gky234).
177. Baldi, Sandro, et al., Genome-wide Rules of Nucleosome Phasing in Drosophila. __Mol Cell__ (2018). [Link](https://doi.org/doi:10.1016/j.molcel.2018.09.032).
178. Chatterjee, Shatakshee, et al., Pre-operative progesterone benefits operable breast cancer patients by modulating surgical stress. __Breast Cancer Res Treat__ (2018). [Link](https://doi.org/doi:10.1007/s10549-018-4749-3).
179. Chen, Yu, et al., PHLDA1, another PHLDA family protein that inhibits Akt. __Cancer Sci__ (2018). [Link](https://doi.org/doi:10.1111/cas.13796).
180. Chishima, Takafumi, et al., Identification of transposable elements contributing to tissue-specific expression of long non-coding RNAs. __Genes__ (2018). [Link](https://doi.org/doi:10.3390/genes9010023).
181. Chèneby, Jeanne, et al., ReMap 2018: An updated atlas of regulatory regions from an integrative analysis of DNA-binding ChIP-seq experiments. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx1092).
182. Cui, Chunmei, et al., Identification and Analysis of Human Sex-biased MicroRNAs. __Genomics Proteomics Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1016/j.gpb.2018.03.004).
183. Dréos, René, et al., MGA repository: A curated data resource for ChIP-seq and other genome annotated data. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx995).
184. Egashira, Yoshihiro, et al., Development of lentiviral vectors for efficient glutamatergic-selective gene expression in cultured hippocampal neurons. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-33509-5).
185. Ferris, Elliott, et al., Accelerated Evolution in Distinctive Species Reveals Candidate Elements for Clinically Relevant Traits, Including Mutation and Cancer Resistance. __Cell Rep__ (2018). [Link](https://doi.org/doi:10.1016/j.celrep.2018.02.008).
186. Fiziev, Petko, et al., ChromTime: Modeling spatio-temporal dynamics of chromatin marks. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1485-2).
187. Imrichova, Hana, et al., ChIP-seq meta-analysis yields high quality training sets for enhancer classification. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/388934).
188. Kawaji, Hideya, et al., dirHub: a trackHub configurator with directory structure projection. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/314807).
189. Kawata, Kentaro, et al., Trans-omic Analysis Reveals Selective Responses to Induced and Basal Insulin across Signaling, Transcriptional, and Metabolic Networks. __iScience__ (2018). [Link](https://doi.org/doi:10.1016/j.isci.2018.07.022).
190. Kehl, Tim, et al., REGGAE: A novel approach for the identification of key transcriptional regulators. __BMC Bioinformatics__ (2018). [Link](https://doi.org/doi:10.1093/bioinformatics/bty372).
191. Khetchoumian, Konstantin, et al., Cell-autonomous transcriptional mechanism for enhancement of translation capacity in secretory cells. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/454421).
192. Lareau, Caleb, et al., Interrogation of human hematopoiesis at single-cell and single-variant resolution. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/255224).
193. Mao, Fengbiao, et al., EpiDenovo: A platform for linking regulatory de novo mutations to developmental epigenetics and diseases. __Nucleic Acids Res__ (2018). [Link](https://doi.org/doi:10.1093/nar/gkx918).
194. Misawa, Aya, et al., lncRNA HOTAIR Inhibits Mineralization in Osteoblastic Osteosarcoma Cells by Epigenetically Repressing ALPL. __Calcif Tissue Int__ (2018). [Link](https://doi.org/doi:10.1007/s00223-018-0434-0).
195. Miura, Itaru, et al., DNA methylation of ANKK1 and response to aripiprazole in patients with acute schizophrenia: A preliminary study. __J Psychiatr Res__ (2018). [Link](https://doi.org/doi:10.1016/j.jpsychires.2018.02.018).
196. Mochizuki, Kentaro, et al., SETDB1 is essential for mouse primordial germ cell fate determination by ensuring bmp signaling. __Development__ (2018). [Link](https://doi.org/doi:10.1242/dev.164160).
197. Mourad, Raphaël, et al., Predicting double-strand DNA breaks using epigenome marks or DNA at kilobase resolution. __Genome Biol__ (2018). [Link](https://doi.org/doi:10.1186/s13059-018-1411-7).
198. Nishizawa, Yujiro, et al., Oncogene c-Myc promotes epitranscriptome m6A reader YTHDF1 expression in colorectal cancer. __Oncotarget__ (2018). [Link](https://doi.org/doi:10.18632/oncotarget.23554).
199. Ochsner, Scott, et al., The Signaling Pathways Project: an integrated ‘omics knowledgebase for mammalian cellular signaling pathways. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/401729).
200. Onodera, Yuta, et al., Inflammation-associated MIR-155 activates differentiation of muscular satellite cells. __PLoS One__ (2018). [Link](https://doi.org/doi:10.1371/journal.pone.0204860).
201. Polouliakh, Natalia, et al., Sequence homology in eukaryotes (SHOE): Interactive visual tool for promoter analysis 06 Biological Sciences 0604 Genetics. __BMC Genomics__ (2018). [Link](https://doi.org/doi:10.1186/s12864-018-5101-3).
202. Raffield, Laura M., et al., Common α-globin variants modify hematologic and other clinical phenotypes in sickle cell trait and disease. __PLoS Genet__ (2018). [Link](https://doi.org/doi:10.1371/journal.pgen.1007293).
203. Romero-Cordoba, Sandra L., et al., Loss of function of miR-342-3p results in MCT1 over-expression and contributes to oncogenic metabolic reprogramming in triple negative breast cancer. __Sci Rep__ (2018). [Link](https://doi.org/doi:10.1038/s41598-018-29708-9).
204. Swoboda, Alexander, et al., STAT3 promotes melanoma metastasis by CEBP-induced repression of the MITF pigmentation pathway. __bioRxiv__ (2018). [Link](https://doi.org/doi:10.1101/422832).
205. Wang, Huabo, et al., Myc and ChREBP transcription factors cooperatively regulate normal and neoplastic hepatocyte proliferation in mice. __J Biol Chem__ (2018). [Link](https://doi.org/doi:10.1074/jbc.RA118.004099).
206. Chen, Yuting, et al., Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development. __Genome Biol__ (2017). [Link](https://doi.org/doi:10.1186/s13059-017-1366-0).
207. Govaere, O., et al., The PDGFRα-laminin B1-keratin 19 cascade drives tumor progression at the invasive front of human hepatocellular carcinoma. __Oncogene__ (2017). [Link](https://doi.org/doi:10.1038/onc.2017.260).
208. Guoa, Michael H., et al., Comprehensive population-based genome sequencing provides insight into hematopoietic regulatory mechanisms. __Proc Natl Acad Sci U S A__ (2017). [Link](https://doi.org/doi:10.1073/pnas.1619052114).
209. Ishigaki, Kazuyoshi, et al., Polygenic burdens on cell-specific pathways underlie the risk of rheumatoid arthritis. __Nat Genet__ (2017). [Link](https://doi.org/doi:10.1038/ng.3885).
210. Kalender Atak, Zeynep, et al., Identification of cis-regulatory mutations generating de novo edges in personalized cancer gene regulatory networks. __Genome Med__ (2017). [Link](https://doi.org/doi:10.1186/s13073-017-0464-7).
211. Kehl, Tim, et al., RegulatorTrail: A web service for the identification of key transcriptional regulators. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkx350).
212. Matsuda, Kazunari, et al., ChIP-seq analysis of genomic binding regions of five major transcription factors highlights a central role for ZIC2 in the mouse epiblast stem cell gene regulatory network. __Development__ (2017). [Link](https://doi.org/doi:10.1242/dev.143479).
213. Naderi, Ali, et al., C1orf64 is a novel androgen receptor target gene and coregulator that interacts with 14-3-3 protein in breast cancer. __Oncotarget__ (2017). [Link](https://doi.org/doi:10.18632/oncotarget.17826).
214. Onodera, Yuta, et al., miR-155 induces ROS generation through downregulation of antioxidation-related genes in mesenchymal stem cells. __Aging Cell__ (2017). [Link](https://doi.org/doi:10.1111/acel.12680).
215. Sanosaka, Tsukasa, et al., DNA Methylome Analysis Identifies Transcription Factor-Based Epigenomic Signatures of Multilineage Competence in Neural Stem/Progenitor Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.08.086).
216. Tanegashima, Kosuke, et al., Epigenetic regulation of the glucose transporter gene Slc2a1 by β-hydroxybutyrate underlies preferential glucose supply to the brain of fasted mice. __Genes Cells__ (2017). [Link](https://doi.org/doi:10.1111/gtc.12456).
217. Turrell, Frances K., et al., Lung tumors with distinct p53 mutations respond similarly to p53 targeted therapy but exhibit genotype-specific statin sensitivity. __Genes Dev__ (2017). [Link](https://doi.org/doi:10.1101/gad.298463.117).
218. Umeyama, Taichi, et al., DMS-Seq for In Vivo Genome-wide Mapping of Protein-DNA Interactions and Nucleosome Centers. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.035).
219. Yevshin, Ivan, et al., GTRD: A database of transcription factor binding sites identified by ChIP-seq experiments. __Nucleic Acids Res__ (2017). [Link](https://doi.org/doi:10.1093/nar/gkw951).
220. Yoshihara, Masahito, et al., Hotspots of De Novo Point Mutations in Induced Pluripotent Stem Cells. __Cell Rep__ (2017). [Link](https://doi.org/doi:10.1016/j.celrep.2017.09.060).
221. Sugeno, Naoto, et al., α-Synuclein enhances histone H3 lysine-9 dimethylation and H3K9me2-dependent transcriptional responses. __Sci Rep__ (2016). [Link](https://doi.org/doi:10.1038/srep36328).
